libraries/core-react/src/TextField/Icon/Icon.tsx

```diff
@@ -38,7 +38,7 @@ const StyledIcon = styled.div`
   ${Variation}
 `
 
-type Props = {
+type TextfieldIconProps = {
```

Comment:
I suppose we don't need to export internal types right?

Reply:
I suppose not as they are not used externally?

Reply:
Hopefully 😓